.github/workflows/python-push.yml

```diff
@@ -56,39 +56,39 @@ jobs:
       run: |
         make develop
     - name: Setup pre-commit
-      if: ${{ (matrix.os == 'ubuntu-latest' && matrix.python-version == '3.8') }}
+      if: ${{ (matrix.os == 'ubuntu-latest' && matrix.python-version == '3.10') }}
```

Comment:
I would suggest defining env vars for the python version: https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

Reply:
Good suggestion. The change for upgrade simply follows the existing pattern.

I opened new issue here: [define env vars for the python version in the github workflows #1559](https://github.com/oscal-compass/compliance-trestle/issues/1559)